### PR TITLE
Issue 4348 - Add tests for dsidm

### PR DESCRIPTION
**Pull request by bsimonova**
Description:
Created test for dsidm initialise option.

Relates: https://github.com/389ds/389-ds-base/issues/4348

Reviewed by: ???

### CODE REVIEW

**Review by droideck — APPROVED** [review]
LGTM!

File: dirsrvtests/tests/suites/clu/dsidm_init_test.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+@pytest.mark.ds4281
+@pytest.mark.skipif(ds_is_older("1.4.2"), reason="Not implemented")
+def test_dsidm_initialise(topology_no_sample):
+    """Check that keep alive entries are created when initializing a master from another one
+
+    :id: eefb59fc-4fdd-4d68-b6c4-b067eb52e881
+    :setup: Standalone instance
+    :steps:
+        1. Create instance without sample entries
+        2  Check there are no sample entries
```
Comment (vashirov) [inline dirsrvtests/tests/suites/clu/dsidm_init_test.py]: Missing dots after digits in steps make RST parser sad :(

**Review by vashirov — CHANGES_REQUESTED** [review]

File: dirsrvtests/tests/suites/clu/dsidm_init_test.py (hunk 1 above)
Comment (bsimonova) [reply dirsrvtests/tests/suites/clu/dsidm_init_test.py]: Oh, right, good catch :D Fixing.

**Review by vashirov — APPROVED** [review]

File: dirsrvtests/tests/suites/clu/dsidm_init_test.py (hunk 1 above)
Comment (vashirov) [reply dirsrvtests/tests/suites/clu/dsidm_init_test.py]: Awesome, thanks! :)

**Pull request merged**